src/org/opendatakit/briefcase/util/BriefcaseVersionManager.java

```diff
@@ -0,0 +1,37 @@
+package org.opendatakit.briefcase.util;
+
+import static org.opendatakit.briefcase.reused.http.RequestBuilder.url;
+
+import java.net.URL;
+import org.opendatakit.briefcase.reused.http.Http;
+import org.opendatakit.briefcase.reused.http.RequestBuilder;
+
+public class BriefcaseVersionManager {
+  private final Http http;
+  private final String current;
+
+  public BriefcaseVersionManager(Http http, String current) {
+    this.http = http;
+    this.current = current.contains("-") ? current.substring(0, current.indexOf("-")) : current;
```

Comment:
Hey, this expression with a conditional operator reads just like English. Is my style influencing you, or is this a slip-up? ;-)

Reply:
Yes! All your help and reviews are super valuable :)

Reply:
I believe the second sentence, but not the first. :-) You would never deliberately switch away from your
```
? something
: something
```

Reply:
Hahaha I normally break lines only if they get too long